components/google_cloud/actions/list-buckets/list-buckets.mjs

```diff
@@ -0,0 +1,19 @@
+import googleCloud from "../../google_cloud.app.mjs";
+
+export default {
+  name: "List Buckets",
+  version: "0.0.1",
+  key: "google_cloud-list-buckets",
+  type: "action",
+  description: "List GCS buckets",
```

Comment:
Could you please add a link to the docs here?. Can you add what `GCS` means to the description please?

Reply:
done